src/demos/calcite-pick-list.html

```diff
@@ -223,13 +235,13 @@ <h2>RTL</h2>
         </calcite-pick-list-item>
       </calcite-pick-list>
       <script>
-        const pickList = document.querySelector("#one");
+        const pickList4 = document.querySelector("#one_rtl");
         pickList.addEventListener("calcitePickListSelectionChange", (event) => {
           console.log(event.detail);
         });
       </script>
 
-      <calcite-pick-list id="two">
+      <calcite-pick-list id="two_rtl">
         <calcite-pick-list-item text-heading="ID" text-description="This is an id for this field and people use it for things." value="ID">
           <calcite-action slot="secondaryAction" id="action-test" label="click-me" onClick="console.log('clicked');">
```

Comment:
sand: aXe is complaining about these ids not being unique.

Reply:
@asangma that was part of your PR for styling and updating the demos. Could you update these ids?